.gitignore

```diff
@@ -21,3 +21,10 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+repository/wso2ei-6.3.0.zip
+target/
+.idea/
+
+org.wso2.carbon.esb.connector.cerner.iml
+repository/sandboxcernercom.crt
```

Comment:
Shall we add file types here like *.iml and *.crt. Also similarly do for server zip.